bse/bsesoundfontrepo.cc

```diff
@@ -387,10 +277,6 @@ SoundFontRepoImpl::~SoundFontRepoImpl ()
       delete_fluid_settings (fluid_settings);
       fluid_settings = NULL;
```

Comment:
It'd be better to merge the above two lines into `delete_fluid_settings (&fluid_settings);` where `delete_fluid_settings` automatically assigns `NULL` to the pointer passed in.

Reply:
`delete_fluid_settings` is part of the fluid synth api, so I cannot change what it does